playbooks/roles/caasp4/tasks/setup_nodes.yml

```diff
@@ -34,14 +34,6 @@
   include_tasks: _setup_nodes_terraform_import_resources.yml
   when: import_terraform_resources|bool
 
-- debug:
```

Comment:
You added theses lines in the previous commit. Can you just not add the lines there instead of adding/removing it?

Reply:
:+1: